release.yaml

```diff
@@ -181,13 +185,15 @@ helm-charts:
   metal-stack:
     metal-control-plane:
       repository: https://helm.metal-stack.io
-      version: 0.3.18
+      version: 0.3.20
   monitoring:
     kube-prometheus-stack:
       repository: https://prometheus-community.github.io/helm-charts
       version: 42.1.0
   third-party:
     meilisearch:
+      image_name: getmeili/meilisearch
+      image_tag: v1.1.1
```

Comment:
This should go beneath docker-images because this is not a helm chart.

Reply:
We decided to revert this in the next release, we leave the optional override in metal-roles but the version should ship with their chart.